web/client/components/misc/Loader.jsx

```diff
@@ -13,6 +13,6 @@ const sizeToClass = size => size > 100
         : 'small';
 const React = require('react');
 module.exports = ({ size, style = {}, className}) => (<div className={className}
-    style={{ width: size, height: size, overflow: "hidden", ...style }}>
+    style={{ width: `${size}px`, height: `${size}px`, overflow: "hidden", ...style }}>
```

Comment:
Note: I removed this fix to test this.

I was checking how this fixed the loader issue and I noticed this in [Catalog.jsx](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/components/catalog/Catalog.jsx#L191) component:

loader use prop size as a string

```jsx
... <Loader size="176" /> ...
```

this probably is applied as is in the new react so it creates a invalid css style (eg: `width: 176;` )

While if we use a number for size prop:
```jsx
... <Loader size={176} /> ...
```

this will be converted to a string in the style with the correct unit of measure (eg: `width: 176px;` ).

I think we should change the prop in Catalog loader instead to change this component.

Reply:
in addition to @allyoucanmap  i would also add propType validation for the loader comp